client/components/domains/featured-domain-suggestions/index.jsx

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { pickBy, includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import DomainRegistrationSuggestion from 'components/domains/domain-registration-suggestion';
+
+export class FeaturedDomainSuggestions extends Component {
+	static propTypes = {
+		cart: PropTypes.object,
+		primarySuggestion: PropTypes.object,
+		secondarySuggestion: PropTypes.object,
+	};
+
+	getChildProps() {
+		const childKeys = [
+			'cart',
+			'isSignupStep',
+			'selectedSite',
+			'domainsWithPlansOnly',
+			'query',
+			'onButtonClick',
+		];
+		return pickBy( this.props, ( value, key ) => includes( childKeys, key ) );
```

Comment:
Am I missing something obvious here or would this be the same as `return pick( this.props, childKeys )`?

Reply:
Ha, you're spot on. I forgot about using `pick.